nginx-fem-redirects.conf

```diff
@@ -1,4 +1,23 @@
 set $fe_project_uri "https://fe-project.zooniverse.org";
+
+location ~* ^/(?:projects|about)/_next/.+?$ {
+    resolver 1.1.1.1;
+    proxy_pass $fe_project_uri;
+    proxy_set_header Host fe-project.zooniverse.org;
+}
+
+location ~* ^/(?:projects|about)/assets/.+?$ {
```

Comment:
If these need to server specific static assets at the target services then they'll need to be split out.

Reply:
👍